### ISSUE TITLE
https://forum.flux.party/ issue

### ISSUE THREAD

**Issue by jamesray1** [voteflux/flux-website-v2#122]
When I went to https://forum.flux.party/ I saw this:

![Flux_forum_invalid_cert-Screenshot_20190531_134523](https://user-images.githubusercontent.com/16969914/58681244-fcc7a000-83ae-11e9-856e-f03677af2c25.png)

However, I can see it, when I don't use https and just go to forum.flux.party. The date and time on my computer are correct.

![Flux_forum_invalid_cert-Screenshot_20190531_134523](https://user-images.githubusercontent.com/16969914/58681445-c76f8200-83af-11e9-8852-c8cf130d3838.png)

![Uploading forum_flux_party_cert_expired-Screenshot_20190531_142144.png…]()

It looks like the certificate expired today.

**Comment by XertroV** [comment]
Yeah - lets encrypt certbot isn't working. But also we set up the forum last year and nobody used it, so I'm not sure it's worth prioritising :(
